kubeflow-pipelines.advisories.yaml

```diff
@@ -200,3 +210,20 @@ advisories:
         data:
           type: vulnerable-code-not-included-in-package
           note: Vulnerable code is part of the Kubernetes ServiceAccount admission plugin which is not importable and also code was marked NOT_IMPORTABLE in Golang vulndb
+
+  - id: CVE-2023-28155
+    aliases:
+      - GHSA-p8p7-x288-28g6
+    events:
+      - timestamp: 2023-11-01T07:12:35Z
+        type: fix-not-planned
```

Comment:
`fix-not-planned` implies that we don't ever expect to see this vulnerability to go away. Given this package isn't pinned to an older version, I'd recommend using the status `pending-upstream-fix`. Upstream in this case refers to the `kubeflow-pipelines` project. That project could eventually migrate away from needing the `request` package.

Reply:
okay that makes sense, thanks @pdeslaur 🤝